aws/ecs-service/main.tf

```
@@ -79,6 +79,27 @@ resource "aws_ecs_task_definition" "this" {
   tags = local.tags
 }
 
+resource "aws_ecs_task_definition" "job" {
+  family                   = var.service_name
+  network_mode             = "awsvpc"
+  execution_role_arn       = aws_iam_role.ecs_execution_role.arn
+  task_role_arn            = aws_iam_role.ecs_task_role.arn
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = var.cpu
+  memory                   = var.memory
+
+  container_definitions = jsonencode([local.job_container_definition])
```

In `locals.tf` this is plural:

```suggestion
  container_definitions = jsonencode([local.job_container_definitions])
```

---

changed the local as there is only one container